neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
```

Comment:
I'd prefer to not add a dependency on matplotlib - is there any way to achieve the same effect without this?